### PR TITLE
fix: always generate rpm .repo file

### PR DESCRIPTION
**Pull request by rubenruizdegauna**
Don't rely on .repo file in repository. Always recreate with specified `accessPointHost`.

### CODE REVIEW

**Review by varas — APPROVED** [review]

File: publisher/publisher.go [hunk 1]
```diff
@@ -483,16 +483,12 @@ func uploadRpm(conf config, srcTemplate string, upload Upload, arch string) (err
...
-			l.Println(fmt.Sprintf("creating 'newrelic-infra.repo' file in %s", s3RepoPath))
-
-			repoFileContent := generateRepoFileContent(conf.accessPointHost, destPath)
-
-			err := ioutil.WriteFile(s3DotRepoFilepath, []byte(repoFileContent), 0644)
-			if err != nil {
-				return err
-			}
+		// create .repo file
+		l.Println(fmt.Sprintf("creating 'newrelic-infra.repo' file in %s", s3RepoPath))
```
Comment (brushknight) [inline publisher/publisher.go:487]: Don't forget about this piece `conf.accessPointHost != ""`

Comment (brushknight) [inline publisher/publisher.go:487]: Don't forget about this piece `conf.accessPointHost != ""`

**Review by brushknight — CHANGES_REQUESTED** [review]

File: publisher/publisher.go (hunk 1 above)
Comment (rubenruizdegauna) [reply publisher/publisher.go:487]: I thought about it but cannot be empty anymore. I can still leave the `if` if you think is safer :)

Comment (brushknight) [reply publisher/publisher.go:487]: good, if empty, then default, right?

Comment (rubenruizdegauna) [reply publisher/publisher.go:487]: yes, fallback to prod

**Review by brushknight — APPROVED** [review]

**Pull request merged**